brambling/models.py

```diff
@@ -482,17 +551,44 @@ def get_absolute_url(self):
     def get_liability_waiver(self):
         return self.liability_waiver.format(event=self.name, organization=self.organization.name)
 
-    def editable_by(self, user):
-        return (
-            self.organization.editable_by(user) or (
-                user.is_authenticated() and
-                user.is_active and
-                self.additional_editors.filter(pk=user.pk).exists()
+    def get_permissions(self, person):
```

Comment:
This function feels slightly unwieldy, with the potential to become more unwieldy if we ever add more permissions, more roles, or more membership types (or need to change some of the above). I'm not sure I have any super-good ideas about what to do instead, which is frustrating because all that's really going on here is taking, like, the union of { OrganizationMemberPermissions, EventMemberPermissions, SuperuserPermissions }.

I might need to ponder this more, but I think it's worth pondering. I don't want us to get into a situation where it's hard to tell who has permission to do what, which I think this might be on the verge of.